### DISCUSSION TITLE
How to prevent link target from being removed?

### DISCUSSION THREAD

**Discussion by jrodiger-lifemine** [GMOD/jbrowse-components#2982] [Q&A] [answered]
When including links in GFF3 track attributes the link target is being removed which is preventing the link from automatically opening in a new tab as we're intending. Is there any way to change this behavior? For example:

`<a href="https://example.com" target="_blank">example link</a>`

is converted to

`<a href="https://example.com">example link</a>`

Thanks!

**Comment by cmdcolin** [comment, marked as answer]
Hi there, I added a small PR that might be able to help this https://github.com/GMOD/jbrowse-components/pull/2983

**Reply by cmdcolin** [reply]
If you have any technical opinion on the PR feel free to comment :) Can also test it the branch build if you are using jbrowse web using `jbrowse create --branch include_target_blank newfolder` from the CLI

**Comment by cmdcolin** [comment]
@jrodiger-lifemine in the latest release v1.7.10, we now actually automatically make all links use _target=blank, and also have the ability to create callbacks for customized feature details! here are some release notes https://jbrowse.org/jb2/blog/2022/06/13/v1.7.10-release/

**Reply by jrodiger-lifemine** [reply]
Awesome thank you!
